magic-modules/make-comment.sh

```diff
@@ -0,0 +1,28 @@
+#!/bin/bash
+
+set -e
+set -x
+
+shopt -s dotglob
+cd mm-initial-pr
+ID=$(git config --get pullrequest.id)
+BRANCH=$(git config --get pullrequest.branch)
+echo $BRANCH
```

Comment:
If these goes to log you may want to have more formatted output `echo "Branch: ${BRANCH}"`, etc.

ditto throughout

Reply:
Those are leftover from when they were helpful for debugging ... but then I just added `set -x` and now debugging is _really_ easy.  Removed 'em.